src/source/tealiumBuilder.brs

```diff
@@ -53,17 +54,32 @@ function TealiumBuilder(account as String, profile as String, logLevel = 3 as In
             end if
         end function
 
+        _IsValidTraceId: function () as Boolean
+            'when defined, only valid if doesn't contain empty spaces, and not an empty string
+            if m._traceId <> invalid
+                return m._traceId.Instr(0, " ") = -1 and m._traceId <> ""
+            else
+                'traceId is optional, so still valid if not defined
+                return true
+            end if
+        end function
+
         _IsValidConfig: function () as Boolean
             'valid if account, profile, datasource, and environment are all valid.
             'datasource and environment are valid if they are not defined since they are optional
-            return m._IsValidAccount() and m._IsValidProfile() and m._IsValidDatasource() and m._IsValidEnvironment()
+            return m._IsValidAccount() and m._IsValidProfile() and m._IsValidDatasource() and m._IsValidEnvironment() and m._IsValidTraceId()
```

Comment:
Are we sure we want to stop the TealiumCore build if the `traceId` is malformed? 
It should only be used while debugging the platform, so it's probably worth ignoring these errors.